pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -254,16 +259,6 @@ func SetDefaults_KubeSchedulerConfiguration(obj *KubeSchedulerConfiguration) {
 		obj.MetricsBindAddress = net.JoinHostPort("0.0.0.0", strconv.Itoa(ports.SchedulerPort))
 	}
 
-	if len(obj.ClientConnection.ContentType) == 0 {
```

Comment:
I didn't expect these per-component defaults go away

Reply:
Does in Line #274?

Reply:
That calls RecommendedDefaultClientConnectionConfiguration which might happen to match right now, but I think we want component specific qps/burst defaults. The defaults for the scheduler, kubelet, and controller manager are likely to be different

Reply:
Yes, notice it before :+1: 
As comment in https://github.com/kubernetes/kubernetes/pull/66993#pullrequestreview-145902542, we add default value for `kube-controller-manager` and `cloud-controller-manager` in owner `SetDefault_Foo`. I think, we can set `how to fix the issuse` as folllow-up.

Reply:
May be we can split the `types.go` and `defaults.go` in different packages, so that we can set specific qps/burst defaults for different component. like that one: https://github.com/kubernetes/kubernetes/tree/master/pkg/apis/batch/v1, `defaults.go` in here, but the `types.go` in `k8s.io/api/batch/v1`.

Reply:
fixed this by adding the `if zeroValue { default }` statements back for qps and burst which is component-specific.